frontend/src/pages/Author.jsx

```diff
@@ -92,7 +93,7 @@ class Author extends Component {
 
     getFriendsPane() {
 	    return (
-	        <Tab.Pane>Friend List component goes here</Tab.Pane>
+	        <Tab.Pane><Friends/></Tab.Pane>
```

Comment:
pass this.props.location.state.fullAuthorId into Friends component as a prop, this will indicate whose profile we are viewing right now